<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="btn_add_course">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
color: rgb(255, 255, 255);
background-color: rgb(39, 78, 117);
border-radius:6;
}

QPushButton:hover{
color: rgb(255, 255, 255);
background-color: rgb(78, 39, 117);
border-radius:6;
}</string>
   </property>
   <property name="text">
    <string>Add Course</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(56, 56, 56);</string>
   </property>
   <property name="text">
    <string>Course List</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>45</y>
     <width>790</width>
     <height>3</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>780</width>
     <height>500</height>
    </rect>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>line</zorder>
  <zorder>listWidget</zorder>
  <zorder>btn_add_course</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
